Header_CPP.cpp

```diff
@@ -0,0 +1,96 @@
+#include "Task II - Header.h"
+int Store::getVecSize()
+{
+	return itemIDs.size();
+}
+void Store::ChngePrices(int a, double b)

```

Comment:
Method names shouldn't be capitalized.

By what the method is doing, I'd rename this one to `setItemSellPrice`.

It's good to be descriptive with the argument names, too.  I'd rename `a` to `itemNumber` and `b` to `sellPrice`.